sn_client/src/register.rs

```diff
@@ -218,10 +228,11 @@ impl ClientRegister {
     // Create a new `ClientRegister` instance with the given name and tag.
     fn new(client: Client, name: XorName, tag: u64) -> Result<Self> {
         let public_key = client.signer_pk();
-        let owner = User::Key(public_key);
-        let perms = Permissions::new_anyone_can_write();
+        // NB TODO permissions should be configurable
+        // using owner only for now, as it is the most restrictive
+        let perms = Permissions::new_owner_only();
```

Comment:
This is not pressing matter but we should, once we get to client work, think about this